tutorials/editor/external_editor.rst

```diff
@@ -65,6 +65,10 @@ Some example **Exec Flags** for various editors include:
     For Emacs, you can call ``emacsclient`` instead of ``emacs`` if
     you use the server mode.
 
+Automatically reloading your changes
+------------------------------------
+To have Godot Editor automatically reload any changes to the scripts you make in an external text editor, enable **Editor > Editor Settings > Text Editor > Behavior > Auto Reload Scripts on External Change**
```

Comment:
```suggestion
To have the Godot Editor automatically reload any script that has been changed by an external text editor,
enable **Editor > Editor Settings > Text Editor > Behavior > Auto Reload Scripts on External Change**.
```
Reads closer to the documentation for the property and easier to parse at least to me, but at minimum:
```suggestion
To have the Godot Editor automatically reload any changes to the scripts you make in an external text editor,
enable **Editor > Editor Settings > Text Editor > Behavior > Auto Reload Scripts on External Change**.
```